src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -256,12 +258,13 @@ protected CloudControllerRestClientImpl() {
     }
 
     public CloudControllerRestClientImpl(URL controllerUrl, CloudCredentials credentials, WebClient webClient, OAuthClient oAuthClient,
-                                         CloudFoundryClient delegate) {
-        this(controllerUrl, credentials, webClient, oAuthClient, delegate, null, null);
+                                         CloudFoundryClient delegate, CloudStackCache cloudStackCache) {
```

Comment:
This object already has too many parameters to its constructor. Adding more isn't great